test/system/test_tpm2_unseal.sh

```diff
@@ -106,7 +106,7 @@ if [ $? != 0 ];then
     exit 1
 fi
 
-tpm2_create -g $alg_create_obj -G $alg_create_key -o $file_unseal_key_pub -O $file_unseal_key_priv -I- -c $file_primary_key_ctx -L $file_policy -A $obj_attr <<< $secret
+tpm2_create -g $alg_create_obj -G $alg_create_key -o $file_unseal_key_pub -O $file_unseal_key_priv -I - -c $file_primary_key_ctx -L $file_policy -A $obj_attr <<< $secret
```

Comment:
As mentioned, I think that this change is not correct since is just papering over the real issue which is that the common options logic is changing the getopt semantics and requiring a space between the option and the argument which shouldn't be needed.